micrometer-core/src/main/java/io/micronaut/configuration/metrics/common/tags/CommonTagsConfiguration.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2017-2019 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.configuration.metrics.common.tags;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+
+import java.util.Map;
+
+/**
+ * The CommonTagsConfiguration will load metrics common tags configuration from application.yml file.
+ */
+@ConfigurationProperties(CommonTagsConfiguration.PREFIX)
+public final class CommonTagsConfiguration {
```

Comment:
Think it makes sense to just add to https://github.com/micronaut-projects/micronaut-micrometer/blob/master/micrometer-core/src/main/java/io/micronaut/configuration/metrics/micrometer/ExportConfigurationProperties.java#L31 rather than add a new configuration properties type